CONTRIBUTING.md

```diff
@@ -26,26 +26,156 @@
+|Workflow status          |status-blocked          |Indicates an issue is blocked on another item of work. The description should be updated with a link to the blocking issue.                                   |                                                                                              |
+|Workflow status          |status-duplicate        |Any issue that duplicates an existing issue.                                                                                                                  |                                                                                              |
+|Workflow status          |status-invalid          |Any issue that is not within scope of the project.                                                                                                            |                                                                                              |
+|Workflow status          |status-wontfix          |Any issue that it is determined will not be fixed.                                                                                                            |                                                                                              |
+|Tracking upstream project|tracking-upstream       |For tracking issues with upstream projects that are affecting the RPCO project.                                                                               |                                                                                              |
+|Product feature          |feature-gating          |Gating-related work.                                                                                                                                          |                                                                                              |
+|Product feature          |feature-upgrades        |Upgrades-related work.                                                                                                                                        |                                                                                              |
+|Product feature          |feature-monitoring      |Monitoring-related work.                                                                                                                                      |                                                                                              |
+|Product feature          |feature-osa             |OSA-related work.                                                                                                                                             |                                                                                              |
+
+## Pre-Backlog -> Backlog
```

Comment:
'Pre-backlog' refers to the state prior to the backlog i.e. before an issue is logged. It is not represented in Waffle because it represents the the point when an issue has not been logged.